src/components/SelectionList/BaseSelectionList.tsx

```diff
@@ -469,7 +467,7 @@ function BaseSelectionList({
                                     onScrollBeginDrag={onScrollBeginDrag}
                                     keyExtractor={(item) => item.keyForList}
                                     extraData={focusedIndex}
-                                    indicatorStyle={theme.white}
+                                    indicatorStyle="white"
```

Comment:
Why was this changed?

Reply:
indicatorStyle accepts only: `'default' | 'black' | 'white' | undefined;` and theme.white is just string